src/core/layout/qgslayoutitemmapoverview.cpp

```diff
@@ -478,6 +478,8 @@ QList<QgsMapLayer *> QgsLayoutItemMapOverviewStack::modifyMapLayerList( const QL
     if ( !l )
       continue;
 
+    l->setCustomProperty( QStringLiteral( "_noset_layer_expression_context" ), true );
```

Comment:
I guess the chances of somebody ever using this exact property name is super low to begin with :)

Reply:
heh. Elsewhere we use '_' prefix for internal keys like this too -- I think that's a fair enough convention to stick with :)